apps/condo/bin/delete-contacts-by-organization-id.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path')
+const { find } = require('@condo/keystone/schema')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { isEmpty } = require('lodash/isEmpty')
+const { Contact } = require('@condo/domains/contact/utils/serverSchema')
+
+class DeleteContactsWithOrganizationId{
+    constructor (organizationIds) {
+        this.organizationIds = organizationIds
+        this.context = null
+    }
+
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
```

Comment:
In the case of some keystone error, this script will fail, because no exceptions are caught. Is it ok?